ocdaction/challenges/views.py

```diff
@@ -27,23 +27,19 @@ def challenge_list(request):
         '-updated_at'
     )[:10]
 
-    for challenge in challenges_not_in_progress:
-        print(challenge.in_progress)
-
     challenges_in_progress_sorted = sorted(
         challenges_in_progress.all(),
         reverse=True,
         key=lambda c: int(c.get_latest_initial_anxiety_level())
     )
 
-    # challenge_in_progress = Challenge.objects.filter(in_progress=True)
-    anxiety_score_card = AnxietyScoreCard.objects.filter(challenge=challenges_in_progress).last()
+    challenge_in_progress = Challenge.objects.filter(in_progress=True)
+    anxiety_score_card = AnxietyScoreCard.objects.filter(challenge=challenge_in_progress).last()
 
```

Comment:
Already have the challenge in progress at the start of the review so line 36 not required